tests/trainer/test_dataloaders.py

```diff
@@ -374,7 +374,7 @@ def test_inf_train_dataloader(tmpdir, check_interval):
 
 
 @pytest.mark.parametrize('check_interval', [50, 1.0])
-@pytest.mark.skip('TODO: speed up this test')
+# @pytest.mark.skip('TODO: speed up this test')
 def test_not_implemented_error_train_dataloader(tmpdir, check_interval):
```

Comment:
this and the following test in which your removed the skip are taking too long because the model runs for the whole epoch, right? But the test suggests that there should be a test for a raise. 
I think that's the problem.

Reply:
we can also set just nb steps to 3 or so...